power_grid_model_c/power_grid_model_c/include/power_grid_model_c/dataset.h

```diff
@@ -147,6 +147,18 @@ PGM_API void PGM_dataset_const_add_buffer(PGM_Handle* handle, PGM_ConstDataset*
                                           PGM_Idx elements_per_scenario, PGM_Idx total_elements, PGM_Idx const* indptr,
                                           void const* data);
 
+/**
+ * @brief Add a attribute buffer to an instance of PGM_ConstDataset/component.
+ * @param handle
+ * @param dataset The pointer to the PGM_ConstDataset.
+ * @param component The name of the component.
+ * @param attribute The name of the attribute.
+ * @param data A void pointer to the buffer data.
+ * @return
+ */
+PGM_API void PGM_dataset_const_add_attribute_buffer(PGM_Handle* handle, PGM_ConstDataset* dataset,
```

Comment:
Please also extend the documentation on the `*_(add|set)_buffer` (particularly that nullptr is now supported)

Reply:
Addressed in the latest commit. Comment resolved.